pkg/rtc/mediatrack.go

```diff
@@ -206,9 +206,10 @@ func (t *MediaTrack) AddReceiver(receiver *webrtc.RTPReceiver, track *webrtc.Tra
 		for _, pkt := range pkts {
 			switch pkt := pkt.(type) {
 			case *rtcp.SourceDescription:
-			// do nothing for now
 			case *rtcp.SenderReport:
-				buff.SetSenderReportData(pkt.RTPTime, pkt.NTPTime)
+				if pkt.SSRC == uint32(track.SSRC()) {
+					buff.SetSenderReportData(pkt.RTPTime, pkt.NTPTime)
+				}
 			case *rtcp.ExtendedReport:
 			rttFromXR:
 				for _, report := range pkt.Reports {
```

Comment:
@cnderrauber Not sure if these reports also should be filtered.

Reply:
No, I intentionally don't check the ssrc as XR can calculate timestamp don't rely on any state so it is safe to get RTT from first DLRR report., and chrome response XR with a strange rule that the media ssrc includes in the SenderSSRC not the DLRR report's SSRC.